rl_loop/distributed_flags_nr

```diff
@@ -5,16 +5,19 @@
 --conv_width=256
 --fc_width=256
 --trunk_layers=19
+--use_SE
```

Comment:
Why are these being specified? I don't think this code calls python?

Reply:
consistency i guess 🙃